tools/runners/yosys_slang.py

```diff
@@ -26,13 +26,27 @@ def get_mode(self, params):
         unsynthesizable = int(params['unsynthesizable'])
         if unsynthesizable:
             return None
+
+        # These tests simply cannot be elaborated because they target
+        # modules that have invalid parameter values for a top-level module,
+        # or have an invalid configuration that results in $fatal calls.
+        name = params["name"]
+        if "black-parrot" in tags and ('bp_lce' in name or 'bp_uce' or 'bp_multicore' in name):
+            return None
+
         return super().get_mode(params)
 
     def prepare_run_cb(self, tmp_dir, params):
         yosys_scr = os.path.join(tmp_dir, "yosys-script")
         mode = params['mode']
 
-        slang_cmd = []
+        slang_cmd += ['-DSYNTHESIS']
+
+        # Ignore content which is unsynthesizable or likely irrelevant for synthesis. The inclusion
+        # of `--ignore-initial` is objectionable as it means the frontend won't pick up memory
+        # initialization.
+        slang_cmd += ['--ignore-timing', '--ignore-initial', '--ignore-assertions']
```

Comment:
It's debatable whether `--ignore-initial` is fair. I'm inclined to push for it since for many applications this is fine, and also the frontend is exposed to many unsynthesizable test cases in this suite (despite the `unsynthesizable` hook from above) so I believe the extent of the frontend's SV support will continue being understated with this flag included. The dashboard results will be more interesting to drive yosys-slang development with these flags in.

Reply:
I should explain: `--ignore-initial` makes the frontend ignore all content of `initial` blocks